libs/application/template-api-modules/src/lib/modules/templates/accident-notification/accident-notification.service.ts

```diff
@@ -49,10 +49,14 @@ export class AccidentNotificationService {
       const answers = application.answers as AccidentNotificationAnswers
       const xml = applictionAnswersToXml(answers, attachments)
 
-      const { ihiDocumentID } = await this.documentApi.documentPost({
+      const { success, ihiDocumentID } = await this.documentApi.documentPost({
         document: { doc: xml, documentType: 801 },
       })
 
+      if (success === 0) {
```

Comment:
does this coincide with the 412 errors your catching here below

Reply:
Nope we can get an error less response with success 0 :/

Reply:
Dont we want to log something here ?